src/connector/src/sink/remote.rs

```diff
@@ -411,145 +402,55 @@ impl<SM, R: RemoteSinkTrait> RemoteSinkWriterInner<SM, R> {
             .start_sink_writer_stream(param.clone(), connector_params.sink_payload_format)
             .await?;
 
-        let schema = param.schema();
-
         Ok(Self {
             properties: param.properties,
             epoch: None,
             batch_id: 0,
             stream_handle,
-            payload_format: connector_params.sink_payload_format,
-            json_encoder: JsonEncoder::new(
-                schema,
-                None,
-                TimestampHandlingMode::String,
-                TimestamptzHandlingMode::UtcWithoutSuffix,
-            ),
```

Comment:
Will we configure the json encoder elsewhere?

Reply:
No. The json format for payload is deprecated in this PR.